lib/src/ufixnum.dart

```diff
@@ -254,6 +254,13 @@ class Register64 {
     }
   }
 
+  void sumReg(Register64 y) {
```

Comment:
Would you mind adding a line of comment to this method?

Also, do you think other digest implementation could benefit from this method?

Reply:
With this tiny change I can merge the PR :)